### PR TITLE
Homework 8

### PR DESCRIPTION
**Pull request by palesya**
Task_8.html file created

### CODE REVIEW

**Review by vshadrin1991 — COMMENTED** [review]
Everything looks good. 
But please add the target directory to .gitignorefile

File: src/test/java/Homework_8/Homework_8.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+public class Homework_8 extends BaseTest {
+
+    @Test(priority = 1)
+    public void checkFirstValueInList() {
+        driver.get("D:/AutomationPesetskaya_QA11/src/test/java/Homework_8/task_8.html");
+        //Создать коллекцию из значений в первой колонке
+        List<WebElement> brandsWebEl = driver.findElements(By.xpath("//tr/td[1]"));
```
Comment (vshadrin1991) [inline src/test/java/Homework_8/Homework_8.java]: Prettier locator for table element would be By.cssSelector(td:first-child)

Comment (vshadrin1991) [reply src/test/java/Homework_8/Homework_8.java]: It's just a note

Comment (palesya) [reply src/test/java/Homework_8/Homework_8.java]: Thanks for advice - already changed

**Comment by palesya** [comment]
Added the target directory and idea folder to .gitignore file

**Pull request merged**